client/src/AppBundle/Service/Client/Sirius/SiriusApiGatewayClient.php

```diff
@@ -100,7 +103,8 @@ public function sendSupportingDocument(SiriusDocumentUpload $upload, string $sub
     {
         $reportJson = $this->serializer->serialize(['supporting_document' => ['data' => $upload]], 'json');
 
-        $this->logger->warning("Syncing supporting document ID with UUID: $submissionUuid");
+        $this->logger->warning("Syncing supporting document with UUID: $submissionUuid");
+        $this->logger->warning($reportJson);
```

Comment:
Just checking if any of these logs are leftover debug?

Reply:
N -, I've found it really useful to be able to see the UUIDs when syncing a document along with what's actually being sent. It also has the plus side of showing what's in each payload so if we get any unexpected errors we can go to integrations with what was sent.